lib/trento/clusters.ex

```diff
@@ -175,6 +198,8 @@ defmodule Trento.Clusters do
           %Checks.ClusterExecutionEnv{
             provider: provider,
             cluster_type: cluster_type,
+            ensa_version:
+              details["sap_systems"] |> Enum.map(fn s -> s["sid"] end) |> get_ensa_version,
```

Comment:
We could make `get_ensa_version` accept the cluster's `details` (or the whole cluster) and encapsulate extraction of the sids of a cluster there, so that here we'd have something like

```elixir
    ensa_version: get_ensa_version(details), # or get_ensa_version(cluster)
```

Another option might be having a private function for sids extraction from a cluster so that here we'd have something like
```elixir
    ensa_version: cluster |> get_sids |> get_ensa_version
```

Not a strong requirement, just sharing thoughts. I am also fine with current proposal. Up to you.